packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -585,19 +585,23 @@ export class AtomicSearchBox {
     lastIndex: number,
     side: 'left' | 'right'
   ) {
-    const id = `${this.id}-${side}-suggestion-${item.key}`;
+    const id = this.getSuggestionId(side, item);
+
     const isSelected = id === this.activeDescendant;
```

Comment:
how about 
```js
    const isSelected =
      id === this.activeDescendant ||
      (this.suggestedQuery === item.query &&
        !this.panelInFocus?.getAttribute('part')?.includes(side));
```
?
removes the need for the extra linking stuff and also supports swapping the sides if we ever want to do that

Reply:
Great suggestion !